src/features/search/components/SearchDropdown.tsx

```
@@ -110,80 +144,206 @@ const SearchDropdown: React.FC<SearchDropdownProps> = ({
-                        ? 'text-emerald-700'
-                        : 'text-gray-800'
+                      'cursor-pointer px-3 py-3 flex items-start gap-2 transition-all duration-150 border-l-3',
+                      isHighlighted
+                        ? 'bg-blue-50 border-blue-500 shadow-sm'
+                        : 'hover:bg-gray-50 border-transparent'
                     )}
+                    onClick={() => onSelectItem(description, overallIndex)}
+                    onMouseEnter={() => onHighlightItem(overallIndex)}
+                    role="option"
+                    aria-selected={isHighlighted}
                   >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "option" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "option".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)